src/busctl/busctl.c

```diff
@@ -809,7 +809,7 @@ static int on_interface(const char *interface, uint64_t flags, void *userdata) {
                 return log_oom();
 
         r = set_put(members, m);
-        if (r == -EEXIST)
+        if (r == 0)
                 return log_error_errno(r,  "Invalid introspection data: duplicate interface '%s'.", interface);
```

Comment:
Please do not pass `0` to `log_error_errno()`, but use `SYNTHETIC_ERRNO()` macro.
```
return log_error_errno(SYNTHETIC_ERRNO(EEXIST), "...");
```
Please also drop doubled white space.

Reply:
Thanks, I somehow missed this. Updated.